src/taxes/pages/TaxCountriesPage/TaxCountriesPage.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  Card,
+  CardContent,
+  InputAdornment,
+  TextField
+} from "@material-ui/core";
+import VerticalSpacer from "@saleor/apps/components/VerticalSpacer";
+import CardTitle from "@saleor/components/CardTitle";
+import Container from "@saleor/components/Container";
+import Grid from "@saleor/components/Grid";
+import PageHeader from "@saleor/components/PageHeader";
+import Skeleton from "@saleor/components/Skeleton";
+import {
+  CountryFragment,
+  TaxCountryConfigurationFragment
+} from "@saleor/graphql";
+import { sectionNames } from "@saleor/intl";
+import {
+  List,
+  ListHeader,
+  ListItem,
+  ListItemCell,
+  makeStyles,
+  PageTab,
+  PageTabs,
+  SearchIcon
+} from "@saleor/macaw-ui";
+import { parseQuery } from "@saleor/orders/components/OrderCustomerAddressesEditDialog/utils";
+import { taxesMessages } from "@saleor/taxes/messages";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import TaxInput from "../../components/TaxInput";
+import TaxCountriesMenu from "./TaxCountriesMenu";
+
+interface TaxCountriesPageProps {
+  countryTaxesData: TaxCountryConfigurationFragment[];
```

Comment:
This field should be optional (to match what's used in `loading` story)
```suggestion
  countryTaxesData?: TaxCountryConfigurationFragment[];
```